testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/actions/AppInitiatedActionUpdateProfileTest.java

```diff
@@ -363,12 +363,7 @@ public void updateProfileExpiredCookies() {
         driver.manage().deleteAllCookies();
 
         updateProfilePage.prepareUpdate().username("test-user@localhost").firstName("New first").lastName("New last").email("keycloak-user@localhost").submit();
-        errorPage.assertCurrent();
-
-        String backToAppLink = errorPage.getBackToApplicationLink();
-
-        ClientRepresentation client = ApiUtil.findClientByClientId(adminClient.realm("test"), "test-app").toRepresentation();
-        Assert.assertEquals(backToAppLink, client.getBaseUrl());
+        loginPage.assertCurrent();
```

Comment:
Could you please clarify what page exactly is displayed here? If I understand correctly, there is not "error page" displayed, but there is the "info page" displayed, which still contains the message "Cookie not found" and link "Back to application" .

So if I understand correctly, this PR updates things in a way that instead of "Error page", the "Info page" is shown, but the message and link is pretty much the same as before. Is it correct?